ibm/service/vpc/resource_ibm_is_snapshot.go

```diff
@@ -83,6 +86,80 @@ func ResourceIBMSnapshot() *schema.Resource {
 				Description:  "Snapshot name",
 			},
 
+			isSnapshotSourceSnapshotCRN: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Source Snapshot CRN",
+			},
+
```

Comment:
I see a new field "copies" in the spec.
Also, encryption_key is added to accept key crn.

Reply:
couldn't see isSnapshotEncryptionKey defined in schema that takes input